snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers2.cs

```diff
@@ -1,41 +1,3 @@
-//<Snippet1>
```

Comment:
This also still seems to be in use:

programming-guide\unsafe-code-pointers\pointer-conversions.md: [!code-csharp[csProgGuidePointers#3](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers2.cs#3)]

Reply:
Only `<Snippet3>` is referenced.

Reply:
Again, I should have noticed that, @pkulikov. I apologize.